q2_makarsa/plugin_setup.py

```diff
@@ -1,10 +1,12 @@
 from networkx import Graph, read_graphml, write_graphml
 from q2_types.feature_table import FeatureTable, Frequency
 from qiime2.plugin import Bool, Float, Int, Metadata, Plugin, Str
+import pandas as pd
 
-from ._network import Network, NetworkDirectoryFormat, NetworkFormat
+from ._network import Network, NetworkDirectoryFormat, NetworkFormat, NodeCommunityFormat, Node_Community, NodeDirectoryFormat
```

Comment:
NodeCommunity is rather long and not too intuitive to me. What about `NodeMap` or `NetworkModules` or something like this?